tests/stub-cards/valid-cards/node_modules/article-card/schema.js

```diff
@@ -0,0 +1,39 @@
+const JSONAPIFactory = require('@cardstack/test-support/jsonapi-factory');
+module.exports = function({sourceId, packageName}) {
+  // Article Card Schema
+  let factory = new JSONAPIFactory(sourceId, packageName);
+
+  factory.addResource('card-definitions')
+    .withRelated('model', factory.addResource('content-types', 'articles')
+      .withAttributes({ name: 'articles' })
+      .withRelated('fields', [
+        factory.addResource('fields', 'title').withAttributes({
+          name: 'title',
+          'is-metadata': true,
+          'needed-when-embedded': true,
+          'field-type': '@cardstack/core-types::string'
+        }),
+        factory.addResource('fields', 'body').withAttributes({
+          name: 'body',
+          'is-metadata': true,
+          'field-type': '@cardstack/core-types::string'
+        }),
+        factory.addResource('fields', 'categories').withAttributes({
+          name: 'categories',
+          'is-metadata': true,
+          'needed-when-embedded': true,
+          'field-type': '@cardstack/core-types::has-many'
+        }).withRelated('related-types', [
+          // this is modeling an enumeration
+          factory.addResource('content-types', 'categories').withAttributes({ name: 'categories' })
```

Comment:
same comment around being duplicative for setting the `name` attribute on `content-types` to be the same as the "id" of the resource (which is not the actual ID because the ID is really a composite)